data/map.txt

```diff
@@ -2108,6 +2108,8 @@ system Aescolanus
 	attributes "ember waste" "wormhole"
 	habitable 233.28
 	belt 1323
+	hazard "Ember Waste Base Heat" 100
+	hazard "Ember Waste Base Storm" 9000
```

Comment:
I'd place hazard lines below fleet lines so that the connection between fleet spawns and hazard spawns is easier to make at a glance.

Reply:
Done. Changed.